src/gwt/src/org/rstudio/studio/client/application/DesktopFrame.java

```diff
@@ -127,7 +127,10 @@ void promptForText(String title,
 
    void bringMainFrameToFront();
    void bringMainFrameBehindActive();
-   
+
+   void getRVersion(CommandWithArg<String> callback);
```

Comment:
Maybe add a comment warning that these are currently only implemented for Windows?

Reply:
Those have always been Windows-only, but sure, will add a comment.